src/main/resources/twilio-go/api.mustache

```diff
@@ -78,10 +79,10 @@ func (c *{{{classname}}}Service) {{{nickname}}}({{#allParams}}{{#required}}{{par
 	    return nil, err
 	}
 
-	data.Set("{{paramName}}", string(v))
+	data.Set(html.UnescapeString("{{baseName}}"), string(v))
```

Comment:
Not particularly proud of this change but I couldn't find an easy way to modify the `baseName` in the openapi generator. Open for suggestions!

Reply:
Why is the unescaping needed? Tracing through `DefaultCodegen` and `AbstractCodegen` looks like there may be a way around needing this, but not entirely sure where it's coming from.

[This](https://github.com/OpenAPITools/openapi-generator/blob/032911990d6f7f60a23b2547bbbdc7608fa6b731/modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java#L4195) sets the base name using the parameter name which is set [here](https://github.com/OpenAPITools/openapi-generator/blob/032911990d6f7f60a23b2547bbbdc7608fa6b731/modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java#L3124) using the result of `toVarName` which is overridden [here](https://github.com/OpenAPITools/openapi-generator/blob/032911990d6f7f60a23b2547bbbdc7608fa6b731/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractGoCodegen.java#L190).

Reply:
So, turns out the values returned by the double-stash `{{}}` are html-escaped and if we don't want the escaping, we should be using the triple-stash `{{{}}}`. Updated the code